BusinessLogic/Classes/NAudioPlayer.cs

```diff
@@ -0,0 +1,83 @@
+﻿using BusinessLogic.Enums;
+using NAudio.Wave;
+using NAudio.Wave.SampleProviders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogic.Classes
+{
+    public class NAudioPlayer
+    {
+        public NAudioPlayer()
+        {
+        }
+    
+        public void PlayFrequency(int frequency, float volume, Ear ear)
+        {
+            #region part1
+            //int sampleRate = 44100; // Sample rate in Hz (standard for most audio)
```

Comment:
Is deze uitgecommende code nodig? anders zou ik het weghalen

Reply:
Zal ik weghalen, wilde het bewaren voor het geval de code bij jou of Jasper niet werkte